### ISSUE TITLE
Error message missing in the popup dialog

### ISSUE THREAD

**Issue by ninogjoni** [directus/directus#22420]
### Describe the Bug

An error message should appear in the data studio popup when a flow triggers an error. Nevertheless, it actually shows the error message when I provoke an error with postman.

Data Studio:
![image](https://github.com/directus/directus/assets/49320058/aeef2d65-4654-42f7-adfa-de835452012f)

Postman result:
```
{
    "data": {
        "update_report_strings_item": null
    },
    "errors": [
        {
            "message": "Validation failed: The value 'Hello World' does not match the regex pattern '^[A-HJ-NPR-Z0-9]{7}$'.",
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR"
            },
            "locations": [
                {
                    "line": 2,
                    "column": 5
                }
            ],
            "path": [
                "update_report_strings_item"
            ]
        }
    ]
}
```

### To Reproduce

Create a flow, add a 'Run Script' and implement a throw Error statement.

For example:
```
module.exports = async function(data) {
    throw new Error(`Validation failed: The value '${valueToValidate}' does not match the regex pattern '${regexPattern}'.`);
    return {
        data
    };
}
```

### Directus Version

v10.10.7

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by paescuj** [comment]
Will be fixed by #22379.

**Comment by robinsonjohn** [comment]
Can confirm the same using Directus v10.11.0
